src/main/java/com/example/java/test/junior/developer/controller/UserController.java

```diff
@@ -0,0 +1,50 @@
+package com.example.java.test.junior.developer.controller;
+
+import com.example.java.test.junior.developer.dto.LanguageDto;
+import com.example.java.test.junior.developer.dto.UserDto;
+import com.example.java.test.junior.developer.model.User;
+import com.example.java.test.junior.developer.service.LanguageService;
+import com.example.java.test.junior.developer.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/user")
```

Comment:
Let's follow what we have in Language contoller - so it should be `/api/v1/users`

Reply:
Please - note - userS not user - this is how you should call REST resources.